### PR TITLE
Functional tests - Add new test 'FO > Hummingbird > Add to cart'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add new test 'FO > Hummingbird > Add to cart'
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI and nightly are :green_apple: 
| UI Tests          | https://github.com/nesrineabdmouleh/testing_pr/actions/runs/7961332861
| Fixed issue or discussion?     | no
| Related PRs       | no
| Sponsor company   |

### CODE REVIEW

File: tests/UI/pages/FO/hummingbird/searchResults/index.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 // Import FO pages
 import {SearchResultsPage} from '@pages/FO/classic/searchResults';
+import {Page} from 'playwright';
```
Comment (Progi1984) [inline tests/UI/pages/FO/hummingbird/searchResults/index.ts]:
```suggestion
import type {Page} from 'playwright';
```

File: tests/UI/pages/FO/hummingbird/home/index.ts [hunk 2]
```diff
@@ -16,6 +16,8 @@ class Home extends HomePage {
 
   private readonly addToCartIcon: (number: number) => string;
 
+  public readonly blockCartModalSummary: string;
```
Comment (Progi1984) [inline tests/UI/pages/FO/hummingbird/home/index.ts]:
```suggestion
  private readonly blockCartModalSummary: string;
```

File: tests/UI/pages/FO/hummingbird/cart/index.ts [hunk 3]
```diff
@@ -1,5 +1,6 @@
 // Import FO pages
 import {CartPage} from '@pages/FO/classic/cart/index';
+import {Page} from 'playwright';
```
Comment (Progi1984) [inline tests/UI/pages/FO/hummingbird/cart/index.ts]:
```suggestion
import type {Page} from 'playwright';
```

File: tests/UI/campaigns/functional/FO/hummingbird/09_productPage/01_quickView/01_addToCart.ts [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+import cartPage from '@pages/FO/hummingbird/cart';
+import searchResultsPage from '@pages/FO/hummingbird/searchResults';
+
+// Import data
+import Products from '@data/demo/products';
+import CartProductDetails from '@data/types/cart';
+import {ProductAttribute} from '@data/types/product';
+
+import {expect} from 'chai';
+import {BrowserContext, Page} from 'playwright';
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/FO/hummingbird/09_productPage/01_quickView/01_addToCart.ts]:
```suggestion
import type {BrowserContext, Page} from 'playwright';
```

**Review by Progi1984 — COMMENTED** [review]
Some feedbacks :)

**Review by Progi1984 — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Pull request merged**